### PR TITLE
docs: update k8s-microk8s.md

### PR DESCRIPTION
**Pull request by jenworthington**
Edits and one question towards the end of the topic.

### CODE REVIEW

**Comment by mstopa-splunk** [comment]
@jenworthington please pull my changes and continue working on `k8s-microk8s-new.md` which I will use to overwrite the old file entirely when we finish

File: docs/gettingstarted/k8s-microk8s.md [hunk 1]
```diff
@@ -1,42 +1,34 @@
...
-# Install MicroK8s
-The SC4S deployment model with Microk8s uses specific features of this distribution of k8s. 
-While this may be reproducible with other distributions such an undertaking requires more advanced
-awareness and responsibility for the administrator.
+# Install and configure SC4S with Kubernetes
+Splunk provides an implementation for SC4S deployment with MicroK8s using a single-server MicroK8s as the deployment model. Clustering has some tradeoffs and should be only considered on a deployment-specific basis.
 
-* (metalLB) ensure source IP is preserved
-* Bring any operating system (window/centos/rhel/ubuntu/debian)
+You can independently replicate the model deployment on different distributions of Kubernetes. To do this you must have Administrator privleges, and the process will require advanced understanding and responsibility on your part.
```
Comment (mstopa-splunk) [inline docs/gettingstarted/k8s-microk8s.md:5]:
```suggestion
You can independently replicate the model deployment on different distributions of Kubernetes. To do this you must have Administrator privileges, and the process will require advanced understanding and responsibility on your part.
```

I think the original piece mean that the administrator is expected to have advanced awareness and responsibility, not that Administrator privileges are required

Comment (mstopa-splunk) [inline docs/gettingstarted/k8s-microk8s.md:5]: I think the original piece mean that the administrator is expected to have advanced awareness and responsibility, not that Administrator privileges are required

File: docs/gettingstarted/k8s-microk8s.md [hunk 2]
```diff
@@ -45,82 +37,86 @@ microk8s enable storage
...
-HEC token as a kubernetes secret:
-```bash
-export HEC_TOKEN="00000000-0000-0000-0000-000000000000" # provide your token here!
+export HEC_TOKEN="00000000-0000-0000-0000-000000000000"
 microk8s helm3 install sc4s --set splunk.hec_token=$HEC_TOKEN splunk-connect-for-syslog/splunk-connect-for-syslog -f values.yaml
 ```
-# Upgrade SC4S 
+
+# Update or upgrade SC4S 
+Whenever the image is upgraded or when changes are made to the `values.yaml` file should be applied, run the command:
```
Comment (mstopa-splunk) [inline docs/gettingstarted/k8s-microk8s.md:79]:
```suggestion
Whenever the image is upgraded or when changes are made to the `values.yaml` file and should be applied, run the command:
```

File: docs/gettingstarted/k8s-microk8s.md [hunk 3]
```diff
@@ -45,82 +37,86 @@ microk8s enable storage
...
 ```yaml
 --8<---- "docs/resources/k8s/values_adv_config_file.yaml"
 ```
 
-# Resource Management
+# Manage resources
 
-Generally two instances will be provisioned per node adjust requests and limits to
-allow each instance to use about 40% of each node presuming no other workload is present
+Provision two instances per node. Adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present.
```
Comment (mstopa-splunk) [inline docs/gettingstarted/k8s-microk8s.md:119]: What we mean here is that by default the user should expect two instances per node so they should assign about 40% of resource usage to each instance on every node, unless there is other workload. So maybe sth like:

```suggestion
You should expect two instances per node by default, so adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present.
```

**Comment by mstopa-splunk** [comment]
@jenworthington ready for the next pass

File: docs/gettingstarted/k8s-microk8s.md (hunk 1 above)
Comment (jenworthington) [reply docs/gettingstarted/k8s-microk8s.md:5]: ah! How about something like this? "Do not attempt this unless you have advanced understanding of Kubernetes and are willing and able to maintain this configuration regularly."

File: docs/gettingstarted/k8s-microk8s.md (hunk 3 above)
Comment (jenworthington) [reply docs/gettingstarted/k8s-microk8s.md:119]: How about this: "You should expect your system to require two instances per node by default. Adjust requests and limits to allow each instance to use about 40% of each node, presuming no other workload is present. "

File: docs/gettingstarted/k8s-microk8s.md (hunk 1 above)
Comment (mstopa-splunk) [reply docs/gettingstarted/k8s-microk8s.md]: yess, that's perfect

File: docs/gettingstarted/k8s-microk8s.md (hunk 3 above)
Comment (mstopa-splunk) [reply docs/gettingstarted/k8s-microk8s.md]: great

**Pull request merged**